data/duMasdesBourbouxetal2020/BAO_for_external_use/README

```diff
@@ -0,0 +1,34 @@
+1. The official eBOSS DR16 scans can be found here: https://svn.sdss.org/public/data/eboss/DR16cosmo/tags/v1_0_0/likelihoods/
+
+This is the README for those files:
+The DR16 Lyman-alpha BAO likelihoods are provided separately for the auto (LYAUTO) and cross correlation (LYXQSO). They can be treated as independent. The format for each is:
+
+Column 1:  (DM/rd) at z=2.334
+Column 2:  (DH/rd) at z=2.334
+Column 3:  likelihood (relative to the best point on the grid)
+
+2. Files containing the full scan output with all parameters can be found here: https://github.com/igmhub/picca/tree/master/data/duMasdesBourbouxetal2020. Note those results were run on a smaller grid compared to the official results.
```

Comment:
Which files are these? You should list the file names, otherwise it is confusing. Where these the files used in the Lya BAO paper? If so, say so.

Reply:
Are these HDF5 files? If so, specify it.